mxcubeweb/core/adapter/motor_adapter.py

```diff
@@ -7,6 +11,8 @@
 
 
 class MotorAdapter(ActuatorAdapterBase):
+    SUPPORTED_TYPES: ClassVar[list[object]] = [AbstractMotor.AbstractMotor]
```

Comment:
We define the supported hardware objects like this. And yes RUFF strikes again with the `ClassVar` in-case one would not know that this is a class variable ?